lunes_cms/cms/admins/feedback_admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+class FeedbackAdmin(admin.ModelAdmin):
+    """
+    Admin interface for feedback.
+    Inheriting from `admin.ModelAdmin`.
+    """
+
```

Comment:
```suggestion

    model = Feedback

```
I guess you should define the model here as well.